src/Api/SecretsManager/Controllers/CountsController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Bit.Api.SecretsManager.Models.Response;
+using Bit.Core.Context;
+using Bit.Core.Enums;
+using Bit.Core.Exceptions;
+using Bit.Core.SecretsManager.Repositories;
+using Bit.Core.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Bit.Api.SecretsManager.Controllers;
+
+[Authorize("secrets")]
+[Route("organizations/{organizationId}")]
+public class CountsController : Controller
+{
+    private readonly ICurrentContext _currentContext;
+    private readonly IUserService _userService;
+    private readonly IProjectRepository _projectRepository;
+    private readonly ISecretRepository _secretRepository;
+    private readonly IServiceAccountRepository _serviceAccountRepository;
+
+    public CountsController(
+        ICurrentContext currentContext,
+        IUserService userService,
+        IProjectRepository projectRepository,
+        ISecretRepository secretRepository,
+        IServiceAccountRepository serviceAccountRepository)
+    {
+        _currentContext = currentContext;
+        _userService = userService;
+        _projectRepository = projectRepository;
+        _secretRepository = secretRepository;
+        _serviceAccountRepository = serviceAccountRepository;
+    }
+
+    [HttpGet("counts")]
```

Comment:
Should the route/controller naming be more explicit that it belongs to Secrets Manager?  

Maybe something like `organizations/{organizationId}/sm-counts`.

In a password manager & organization context, I would expect something with this name to return the number of users, groups, collections, etc.

Reply:
Fixed in https://github.com/bitwarden/server/pull/4200/commits/41b166c6059a33ccd52066d0b524f387254ffa0d